### ISSUE TITLE
utPLSQL 3.1.7 is not installable anymore in Oracle EE 11.2.0.4

### ISSUE THREAD

**Issue by tkleiber** [utPLSQL/utPLSQL#959]
**Describe the bug**

Installation shows following compile errors on Oracle EE 11.2.0.4, errors does not happen on EE 19.3.0.0.0:

```
--------------------------------------------------------------
Validating installation
--------------------------------------------------------------
PACKAGE BODY UT_COMPOUND_DATA_HELPER                 1        732         16 PLS-00642: Lokale Erfassungstypen in SQL-Anweisungen nicht zulässig
PACKAGE BODY UT_COMPOUND_DATA_HELPER                 2        732         10 PL/SQL: ORA-22905: Zugriff auf Zeilen eines Objekts, das keine Nested Table ist, nicht möglich
PACKAGE BODY UT_COMPOUND_DATA_HELPER                 3        730          5 PL/SQL: SQL Statement ignored
begin
*
FEHLER in Zeile 1:
ORA-20000: Not all sources were successfully installed.
ORA-06512: in Zeile 3
```

**Provide version info**
Information about utPLSQL and Database version, 
```
11.2.0.4.0
11.2.0.0.0
-
BANNER                                                                          
--------------------------------------------------------------------------------
Oracle Database 11g Enterprise Edition Release 11.2.0.4.0 - 64bit Production
PL/SQL Release 11.2.0.4.0 - Production
CORE	11.2.0.4.0	Production  
TNS for Linux: Version 11.2.0.4.0 - Production
NLSRTL Version 11.2.0.4.0 - Production


PARAMETER                      VALUE                                   
------------------------------ ----------------------------------------
NLS_LANGUAGE                   ENGLISH                                 
NLS_TERRITORY                  GERMANY                                 
NLS_CURRENCY                   €                                       
NLS_ISO_CURRENCY               GERMANY                                 
NLS_NUMERIC_CHARACTERS         ,.                                      
NLS_CALENDAR                   GREGORIAN                               
NLS_DATE_FORMAT                DD.MM.RR                                
NLS_DATE_LANGUAGE              ENGLISH                                 
NLS_SORT                       BINARY                                  
NLS_TIME_FORMAT                HH24:MI:SSXFF                           
NLS_TIMESTAMP_FORMAT           DD.MM.RR HH24:MI:SSXFF                  

PARAMETER                      VALUE                                   
------------------------------ ----------------------------------------
NLS_TIME_TZ_FORMAT             HH24:MI:SSXFF TZR                       
NLS_TIMESTAMP_TZ_FORMAT        DD.MM.RR HH24:MI:SSXFF TZR              
NLS_DUAL_CURRENCY              €                                       
NLS_COMP                       BINARY                                  
NLS_LENGTH_SEMANTICS           BYTE                                    
NLS_NCHAR_CONV_EXCP            FALSE                                   

17 rows selected. 


PORT_STRING                                                 
------------------------------------------------------------
x86_64/Linux 2.4.xx
```

**Information about client software**
Installation via SQL Plus.

**To Reproduce**
1. connect to database EE 11.2.0.4 with sqlplus as dba
2. run @install_headless.sql

**Expected behavior**
Installation should not get compile errors.

**Example code**
-

**Additional context**

**Comment by lwasylow** [comment]
Hi
That will be fixed in 3.1.8 and in current develop. 
It's reported by #947
Long story short for some bizarre reasons the 11.2 XE not failing on compilation and that slipped through our builds. 
Sorry for inconvenience.

**Comment by tkleiber** [comment]
Thanks, tested succesfully with 3.1.8.3126-develop.
